tests/testthat/test-use_r_workflows.R

```diff
@@ -100,3 +100,8 @@ test_that("use_spell_check() works", {
   use_spell_check()
   expect_true(file.exists(".github/workflows/call-spell-check.yml"))
 })
+
+test_that("use_connect_publish() works", {
+  use_connect_publish()
```

Comment:
Thought: the test failed to run locally because this url (https://api.github.com/repos/nmfs-fish-tools/ghactions4r/contents/inst/templates/call-connect-publish.yml?ref=main) could not be found. I think this won't be an issue after merging changes from this branch to main.

Reply:
Yes, great point! If it is still failing after merging to main, something is up.